draft-ietf-tls-tls13.md

```diff
@@ -1517,7 +1517,7 @@ Structure of this message:
            Random random;
            opaque legacy_session_id<0..32>;
            CipherSuite cipher_suites<2..2^16-2>;
-           opaque legacy_compression_methods<1..2^8-1>;
+           opaque legacy_compression_methods[2] = 0x0100;
```

Comment:
I don't think this is a good idea, it is likely to cause issues on servers that receive client hello messages from TLSv1.2 implementations that do enabled compression methods